Misc/NEWS.d/next/Library/2021-08-04-12-29-00.bpo-44822.zePNXA.rst

```diff
@@ -0,0 +1,3 @@
+:mod:`sqlite3` user-defined functions returning :class:`strings <str>` with
+embedded ``NULL`` characters are no longer truncated. Patch by Erlend E.
+Aasland.
```

Comment:
NULL is a pointer in C and special value in SQL. NUL is the ASCII name of character with code 0 (also written as "a null character", with lowercase "null", but NUL is more common in Python documentation).
```suggestion
:mod:`sqlite3` user-defined functions and aggregators returning
:class:`strings <str>` with embedded NUL characters are no longer
truncated. Patch by Erlend E. Aasland.
```